recipe/meta.yaml

```diff
@@ -22,9 +22,9 @@ source:
     - win-msc_ver-1900.patch
     - fix-ABI-issue29943.patch
     - do-not-download-externals.patch  # [win]
-
+    - 0015-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
 build:
-  number: 1
+  number: 3
```

Comment:
@tacaswell I merged #158 so we need this to be 3 to get a new package.

Thanks for the PR BTW!